### PR TITLE
Added count to username if present

### PR DESCRIPTION
**Pull request by Edzelopez**

### CODE REVIEW

File: aoservicelisting.civix.php [hunk 1]
```diff
@@ -354,7 +354,16 @@ public static function generateDrupalUserName($nameDetails) {
     foreach ($nameFields as $field) {
       $nameDetails[$field] = strtolower(preg_replace('/[^a-zA-Z0-9\/\s]+/', '', $nameDetails[$field]));
     }
-    return $nameDetails['first_name'] . '.' . $nameDetails['last_name'] . $nameDetails['id'];
+    // Check if username is present, and append count to it if it is.
+    $query = $con->select('users_field_data', 'u')
```
Comment (seamuslee001) [inline aoservicelisting.civix.php:359]: is $con defined? @Edzelopez ?

**Review by seamuslee001 — CHANGES_REQUESTED** [review]
I think you need to fix the $con variable

File: aoservicelisting.civix.php (hunk 1 above)
Comment (Edzelopez) [reply aoservicelisting.civix.php:359]: Whoops, fixed it.

**Review by seamuslee001 — APPROVED** [review]

**Comment by seamuslee001** [comment]
Looks fine to me now @Edzelopez

**Pull request merged**